examples/scenarios/file.md

```diff
@@ -0,0 +1,44 @@
+# File
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure a file to send logs via Stanza.
+
+## Prerequisites
+
+On the host that will be sending logs to Stanza, make sure you know the location of the file you wish to gather logs from.
```

Comment:
Stanza will be reading logs, not receiving them.

Reply:
Corrected